Source/Editor/Windows/GameWindow.cs

```diff
@@ -39,6 +39,28 @@ public class GameWindow : EditorWindow
         private bool _useAspect = false;
         private bool _freeAspect = true;
 
+        private List<PlayModeFocusOptions> _focusOptions = new List<PlayModeFocusOptions>()
```

Comment:
Update `Name` and `Tooltip` texts to match `Game Window` naming (instead of `Panel`).